lib/ofdm_zkf_c_impl.cc

```diff
@@ -42,6 +40,8 @@ namespace gr {
                              typ_cp));
     }
 
+
```

Comment:
why these whitespaces? (also line 215 of this file)

Reply:
Must have slipped in by mistake.